data/json/recipes/practice/skills.json

```
@@ -0,0 +1,47 @@
+[
+  {
+    "id": "prac_computer_beg",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_SKILLS",
+    "name": "computer (beginner)",
+    "description": "Practice the basics of using a computer",
+    "skill_used": "computer",
+    "time": "30 m",
+    "practice_data": { "min_difficulty": 0, "max_difficulty": 1, "skill_limit": 2 },
+    "autolearn": [ [ "computer", 1 ] ],
+    "book_learn": [ [ "mag_computer", 0 ], [ "howto_computer", 0 ] ],
+    "tools": [ [ [ "laptop", 25 ], [ "eink_tablet_pc", 10 ], [ "smart_phone", 10 ] ] ]
+  },
+  {
+    "id": "prac_computer_int",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_SKILLS",
+    "name": "computer (intermediate)",
+    "description": "Practice using the command line, writing and running scripts making use of common algorithms and data structures",
+    "skill_used": "computer",
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 2, "max_difficulty": 4, "skill_limit": 5 },
+    "autolearn": [ [ "computer", 4 ] ],
+    "book_learn": [ [ "manual_computers", 2 ], [ "computer_science", 3 ], [ "textbook_computer", 3 ] ],
+    "tools": [ [ [ "laptop", 50 ] ] ]
+  },
+  {
+    "id": "prac_computer_adv",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_SKILLS",
+    "name": "computer (advanced)",
+    "description": "Practice coding advanced algorithms and software exploits",
+    "skill_used": "computer",
```

```suggestion
    "skill_used": "computer",
    "skills_required": [ "electronics", 3 ],
```
Does the current code support something like this? If it does, will the practice also give electronics xp?

This isn't necessary on a first pass if you don't already have it, but it would allow another way to distinguish high versus low skill practices.

---

It does support secondary skill requirements, but they wouldn't be strictly required, they'd just add a failure chance (I can change this if desired).
Secondary skills are never trained when crafting.

However... using a computer doesn't require understanding anything about how its transistors work. Also, the kind of hacking that uses the computers skill in game doesn't involve any soldering or circuit bypass, just software. 
The hacking you might be thinking of (bypassing keypads using an electrohack) uses the electronics skill. For that you do need to know about wiring, but there's no computer interaction.

---

Yeah, it's fair to leave electronics out then